### DISCUSSION TITLE
Packaging SQLPage inside Electron/Tauri

### DISCUSSION THREAD

**Discussion by a-gonzalo** [sqlpage/SQLPage#488] [General]
Hello everyone,

I never didn't worked with Electron nor Tauri but, I was asking myself if would be "easy" to pack a web app made with SQLPage. In fact, I know that Electron works with Node.js and Tauri with Rust, so I really guess that with Tauri would be possible.

Have anyone of you tried it?

**Comment by lovasoa** [comment]
Hello ! This is definitely possible, but expect a fair amount of work.

See for instance: https://github.com/tauri-apps/tauri/discussions/2751

Is there a reason why you can't distribute as a web app ?

**Reply by a-gonzalo** [reply]
Hi!
No, there are no reasons. I was just curious to know if anyone did it and how much effort put into it.
I'll check the link, thank you!
